File: hw_06/Python/hw_06/geometry/Cylinder.py
import math

class Cylinder:
    def __init__(self, radius:float=1.0, length:float=1.0):
        self.__radius = radius
        self.__length = length

    def get_radius(self):
        return self.__radius
    
    def set_radius(self, radius:float): 
        self.__radius = radius

    def get_length(self):
        return self.__length
    
    def set_length(self, length:float):
        self.__length = length

    def area(self):
        return math.pi * pow(self.__radius, 2) * (self.__radius + self.__length)
    
    def volume(self):
        return self.area() * self.__length
    
    def cylinder_info(self):

        return f"Cylinder: Radius = {self.__radius:.2f}, Lenght = {self.__length:.2f}, Area = {self.area():.2f}, Volume = {self.volume():.2f}"

File: hw_06/Java/java_oopwork06/src/geometry/Cylinder.java
public class Cylinder {

    private double radius;
    private double length;

    public Cylinder(double radius, double length) {
        this.radius = radius;
        this.length = length;
    }

    public Cylinder() {
        this(1.0, 1.0);
    }

    public double getRadius() {
        return radius;
    }

    public void setRadius(double radius) {
        this.radius = radius;
    }

    public double getLength() {
        return length;
    }

    public void setLength(double length) {
        this.length = length;
    }

    public double area() {
        return 2 * Math.PI * radius * (radius + length);
    }

    public double volume() {
        return area() * length;
    }

    public void showCyInfo() {
        System.out.println("\nCylinder:");
        System.out.printf("Radius = %.2f%n", radius);
        System.out.printf("Length = %.2f%n", length);
        System.out.printf("Area = %.2f%n", area());
        System.out.printf("Volume = %.2f%n", volume());
    }
}


